.github/workflows/release-python.yml

```diff
@@ -62,20 +70,222 @@ jobs:
+        sudo cp $GITHUB_WORKSPACE/openmp-install/lib/libomp.a $DEVELOPER_DIR/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/lib
+
+    - name: vulkansdk for macos
+      if: matrix.os == 'macos-latest'
+      run: |
+        wget https://sdk.lunarg.com/sdk/download/1.3.236.0/mac/vulkansdk-macos-1.3.236.0.dmg?Human=true -O vulkansdk-macos-1.3.236.0.dmg
+        hdiutil attach vulkansdk-macos-1.3.236.0.dmg
+        sudo /Volumes/vulkansdk-macos-1.3.236.0/InstallVulkan.app/Contents/MacOS/InstallVulkan --root $GITHUB_WORKSPACE/vulkansdk-macos-1.3.236.0 --accept-licenses --default-answer --confirm-command install
+        hdiutil detach /Volumes/vulkansdk-macos-1.3.236.0
+
+    - name: Build wheels for macos x86_64
+      if: matrix.os == 'macos-latest' && matrix.arch == 'x86_64'
       uses: pypa/cibuildwheel@v2.16.2
       env:
         CIBW_ARCHS_MACOS: ${{ matrix.arch }}
-        CIBW_ARCHS_LINUX: ${{ matrix.arch }}
-        CIBW_ARCHS_WINDOWS: ${{ matrix.arch }}
         CIBW_BUILD: ${{ matrix.build }}
         CIBW_BUILD_VERBOSITY: 1
-        CIBW_ENVIRONMENT: CMAKE_BUILD_PARALLEL_LEVEL=2
+        CIBW_ENVIRONMENT: CMAKE_BUILD_PARALLEL_LEVEL=2 NCNN_VULKAN=ON VULKAN_SDK=$GITHUB_WORKSPACE/vulkansdk-macos-1.3.236.0/macOS
+          CMAKE_TOOLCHAIN_FILE=$GITHUB_WORKSPACE/toolchains/ios.toolchain.cmake PLATFORM=MAC ARCHS="x86_64"
+          DEPLOYMENT_TARGET="10.9" ENABLE_BITCODE=OFF ENABLE_ARC=OFF ENABLE_VISIBILITY=OFF
+          OpenMP_C_FLAGS="-Xclang -fopenmp" OpenMP_CXX_FLAGS="-Xclang -fopenmp"
+          OpenMP_C_LIB_NAMES="libomp" OpenMP_CXX_LIB_NAMES="libomp"
+          OpenMP_libomp_LIBRARY="$DEVELOPER_DIR/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/lib/libomp.a"
+      with:
+        output-dir: wheelhouse
+
+    - name: Build wheels for macos arm64 and macos universal2
+      if: (matrix.os == 'macos-latest' && matrix.arch == 'arm64') ||
+        (matrix.os == 'macos-latest' && matrix.arch == 'universal2')
+      uses: pypa/cibuildwheel@v2.16.2
+      env:
+        CIBW_ARCHS_MACOS: ${{ matrix.arch }}
+        CIBW_BUILD: ${{ matrix.build }}
+        CIBW_BUILD_VERBOSITY: 1
+        CIBW_ENVIRONMENT: CMAKE_BUILD_PARALLEL_LEVEL=2 NCNN_VULKAN=ON VULKAN_SDK=$GITHUB_WORKSPACE/vulkansdk-macos-1.3.236.0/macOS
+          CMAKE_TOOLCHAIN_FILE=$GITHUB_WORKSPACE/toolchains/ios.toolchain.cmake PLATFORM=MAC_ARM64 ARCHS="arm64"
+          DEPLOYMENT_TARGET="11.0" ENABLE_BITCODE=OFF ENABLE_ARC=OFF ENABLE_VISIBILITY=OFF
+          OpenMP_C_FLAGS="-Xclang -fopenmp" OpenMP_CXX_FLAGS="-Xclang -fopenmp"
+          OpenMP_C_LIB_NAMES="libomp" OpenMP_CXX_LIB_NAMES="libomp"
+          OpenMP_libomp_LIBRARY="$DEVELOPER_DIR/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/lib/libomp.a"
+          Vulkan_INCLUDE_DIR=$GITHUB_WORKSPACE/vulkansdk-macos-1.3.236.0/MoltenVK/include
+          Vulkan_LIBRARY=$GITHUB_WORKSPACE/vulkansdk-macos-1.3.236.0/MoltenVK/dylib/macOS/libMoltenVK.dylib
+      with:
+        output-dir: wheelhouse
```

Comment:
x86_64 package libvulkan.dylib while arm64/universal package libMotenVK.dylib
Can macos x86_64 package really work without moltenvk?

Reply:
Recheck the CI logging, and the repairing process didn't copy libMoltenVK.dylib for macos x86_64 indeed. The Vulkan_INCLUDE_DIR and Vulkan_LIBRARY are specified now in [b7d7e42](https://github.com/Tencent/ncnn/pull/5020/commits/b7d7e42c010dbda1c172532bebd9a34indeedd8cec5a05). The corresponding release test for [b7d7e42](https://github.com/Tencent/ncnn/pull/5020/commits/b7d7e42c010dbda1c172532bebd9a34indeedd8cec5a05) :
https://github.com/Hideousmon/ncnn/actions/runs/6584694265

It's worth noting that the previously built ncnn wheel can function properly with cpu, even without libMoltenVK.dylib on x86_64 macOS. The situation isn't too bad; it's quite promising, haha. 
(https://github.com/Hideousmon/ncnn-gpu-pre-verification/actions/runs/6212630410/job/16862967067 )

Reply:
The ncnn wheel with libMoltenVK.dylib for x86_64 macOS can function properly:
```
(ncnn) zz@zzs-MacBook-Pro ncnnx86_64test % python yolov4-tiny-testbench.py
[0 Intel(R) Iris(TM) Plus Graphics]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[0 Intel(R) Iris(TM) Plus Graphics]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 Intel(R) Iris(TM) Plus Graphics]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 Intel(R) Iris(TM) Plus Graphics]  subgroup=32  basic/vote/ballot/shuffle=1/1/1/1
[0 Intel(R) Iris(TM) Plus Graphics]  fp16-matrix-16_8_8/16_8_16/16_16_16=0/0/0
NCNN Init time: 515.338176 ms
NCNN Inference time: 109.97455200000006 ms
NCNN Detection time: 111.33643299999996 ms
```